### PR TITLE
reviewed docs to fix broken table, images and links

### PR DESCRIPTION
**Pull request by priyanshi-gupta**
### What does this PR do?

A brief description of the change being made with this pull request.
Fixed broken links, images and tables in documentation.

### Motivation

What inspired you to submit this pull request?

### Additional Notes

Anything else we should know when reviewing?

### Review checklist (to be filled by reviewers)

- [ ] Feature or bugfix MUST have appropriate tests (unit, integration, e2e)
- [ ] Make sure you discussed the feature or bugfix with the maintaining team in an Issue
- [ ] Make sure each commit and the PR mention the Issue number (cf the [CONTRIBUTING](CONTRIBUTING.md) doc)

### CODE REVIEW

File: docs/rum_getting_started.md [hunk 1]
```diff
@@ -1,36 +1,58 @@
...
+### Declare SDK as dependency
+
+Declare [dd-sdk-android][1] and [gradle plugin][2] as a dependency in your `build.gradle` file:
+
+```
+   repositories { 
+    maven { 
+      url "https://dl.bintray.com/datadog/datadog-maven" 
+        } 
+      }
```
Comment (xgouchet) [inline docs/rum_getting_started.md:22]: This part of not required anymore and should be removed

File: docs/rum_getting_started.md [hunk 2]
```diff
@@ -1,36 +1,58 @@
...
+
+Declare [dd-sdk-android][1] and [gradle plugin][2] as a dependency in your `build.gradle` file:
+
+```
+   repositories { 
+    maven { 
+      url "https://dl.bintray.com/datadog/datadog-maven" 
+        } 
+      } 
+      dependencies { implementation "com.datadoghq:dd-sdk-android:x.x.x"
```
Comment (xgouchet) [inline docs/rum_getting_started.md:23]: Usually the dependencies block should go after the plugins

**Review by xgouchet — CHANGES_REQUESTED** [review]

File: docs/configure_rum_android_sdk.md [hunk 3]
```diff
@@ -210,12 +200,13 @@ To modify some attributes in your RUM events, or to drop some of the events enti
...
 
-[1]: https://app.datadoghq.com/rum/create
+[1]: https://app.datadoghq.com/rum/application/create
 [2]: /real_user_monitoring/android
 [3]: /real_user_monitoring/android/data_collected
-[4]: /real_user_monitoring/android/configure_android_sdk/track_view
-[5]: /real_user_monitoring/android/configure_android_sdk/track_action
-[6]: /real_user_monitoring/android/configure_android_sdk/track_resource
+[4]: /real_user_monitoring/android/advanced_configuration/#automatically-track-views
+[5]: /real_user_monitoring/android/advanced_configuration/initialization-parameters
```
Comment (kayayarai) [inline docs/configure_rum_android_sdk.md:207]:
```suggestion
[5]: /real_user_monitoring/android/advanced_configuration/#initialization-parameters
```

**Review by kayayarai — CHANGES_REQUESTED** [review]
one tiny missing #

**Review by kayayarai — APPROVED** [review]

File: docs/rum_getting_started.md [hunk 4]
```diff
@@ -1,36 +1,53 @@
...
-    ```conf
+### Declare SDK as dependency
+
+Declare [dd-sdk-android][1] and [gradle plugin][2] as a dependency in your `build.gradle` file:
+
+```
+plugins {
+    id("dd-sdk-android-gradle-plugin")
+}
+    dependencies { implementation "com.datadoghq:dd-sdk-android:x.x.x"
```
Comment (mariusc83) [inline docs/rum_getting_started.md:21]: I think the code format is broken here

**Review by mariusc83 — APPROVED** [review]

File: docs/rum_getting_started.md [hunk 5]
```diff
@@ -1,36 +1,53 @@
...
+### Declare SDK as dependency
+
+Declare [dd-sdk-android][1] and [gradle plugin][2] as a dependency in your `build.gradle` file:
+
+```
+plugins {
+    id("dd-sdk-android-gradle-plugin")
+}
+    dependencies { implementation "com.datadoghq:dd-sdk-android:x.x.x" 
+    }
```
Comment (xgouchet) [inline docs/rum_getting_started.md:22]:
```suggestion
dependencies {
    implementation "com.datadoghq:dd-sdk-android:x.x.x" 
}
```

**Review by xgouchet — CHANGES_REQUESTED** [review]

**Review by xgouchet — APPROVED** [review]

**Pull request merged**